code/modules/mob/living/simple_animal/abnormality/aleph/distortedform.dm

```diff
@@ -526,65 +531,92 @@
 /mob/living/simple_animal/hostile/abnormality/distortedform/proc/DFAttack()
 	if(!can_act)
 		return
+	var/list/target_list = list()
+	for(var/mob/living/L in livinginrange(10, src))
+		if(L.z != z || (L.status_flags & GODMODE))
+			continue
+		if(faction_check_mob(L, FALSE))
+			continue
+		target_list += L
+
 	if(!target)
-		for(var/mob/living/L in livinginrange(10, src))
-			if(L.z != z || (L.status_flags & GODMODE))
-				continue
-			if(faction_check_mob(L, FALSE))
-				continue
-			target = L
-			break
-	if(!target || !ishuman(target) || QDELETED(target))
+		if(LAZYLEN(target_list))
+			target = pick(target_list)
+
+	if(!target || !ishuman(target) || QDELETED(target) || (target_list.len < 2) || prob(50))
 		if(prob(50))
-			DFDonut() //Donut AOE, safe in the middle
+			DFLine() //Safe in the middle, unsafe outside
 		else
-			DFSlam() //Your usual run of the mill AOE
+			DFSlam() //Your usual run of the mill AOE, safe in the middle
 		return
+
 	if(prob(50))
 		DFSpread(target) //Everyone has to stand far away from eachother. or die.
 	else
 		DFStack(target) //Everyone has to bunch up together. or die.
 
-/mob/living/simple_animal/hostile/abnormality/distortedform/proc/DFDonut()
+/mob/living/simple_animal/hostile/abnormality/distortedform/proc/DFLine(combo)
 	can_act = FALSE
 	for(var/turf/L in view(10, src))
```

Comment:
these should probably get added to a list so that it hits the exact same area instead of it being obscurable by smoke or somethin